src/webui/webutils.jl

```diff
@@ -13,43 +13,19 @@ function getcookie(r::HTTP.Request, key::AbstractString, default="")
     return ind === nothing ? default : cookies[ind].value
 end
 
+readtpl(tpl) = Mustache.template_from_file(joinpath(dirname(@__FILE__), "templates", tpl))
```

Comment:
```suggestion
readtpl(tpl::AbstractString) = Mustache.template_from_file(joinpath(@__DIR__), "templates", tpl))
```

Reply:
Also, are you sure you don't want `render_from_file`? I'm not positive myself but this is the exported function with a docstring.